packages/ai-core/src/common/prompt-service.ts

```diff
@@ -68,6 +74,11 @@ export interface PromptService {
      * @param prompt the prompt template to store
      */
     storePrompt(id: string, prompt: string): void;
```

Comment:
as above, do we need this anymore?

Reply:
This would break the API, I don't know whether people are already using it. For conveniance, I would probably leave it.